src/main/java/io/phdata/streamliner/schemadefiner/model/ColumnDefinition.java

```diff
@@ -35,4 +39,62 @@ public ColumnDefinition(String sourceName, String destinationName, String dataTy
         this.dataType = dataType;
         this.comment = comment;
     }
+
+    private String cleanseDataType(String dataType){
+        if(!dataType.endsWith(" IDENTITY")){
+            return dataType.toUpperCase();
+        }else{
```

Comment:
Looks like this project still needs GoogleJavaFormat (I'm a big fan)

Reply:
yes. Java formatter needs to be needed.